crates/diesel_models/src/pm_blocklist.rs

```diff
@@ -0,0 +1,24 @@
+use diesel::{Identifiable, Insertable, Queryable};
+use serde::{Deserialize, Serialize};
+
+use crate::schema::pm_blocklist;
+
+#[derive(Default, Clone, Debug, Eq, Insertable, PartialEq, Serialize, Deserialize)]
+#[diesel(table_name = pm_blocklist)]
+pub struct PmBlocklistNew {
+    pub merchant_id: String,
+    pub pm_hash: String,
+    pub pm_type: String,
```

Comment:
Discuss consistent terminology. Also use an enum instead of String. In the DB as well.
```suggestion
    pub fingerprint_type: String,
```